CHANGELOG.md

```diff
@@ -27,6 +27,15 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
+- oVirt/RHV
+  * Add dynamic inventory.
```

Comment:
It's just one. What you mean are Anisble facts modules for most of the resources.